README.md

```diff
@@ -576,41 +569,21 @@ been created, and an OpenSearch index is created without the appropriate mapping
 This can either be done by calling the `/_cluster/settings` endpoint directly with the
 body:
```

Comment:
This is duplicate-ish of the next line. See the two lines that start with "This can either be done..." in the rendered markdown screenshot:

<img width="1051" alt="image" src="https://github.com/stac-utils/stac-server/assets/7013017/6f2574f1-cb94-419c-b0c6-d6ccb61c3cb3">